### PR TITLE
feat(ui): new date selector design

### PR DESCRIPTION
**Pull request by Chrissy**
date-impliment

### CODE REVIEW

File: src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange.jsx [hunk 1]
```diff
@@ -39,6 +40,11 @@ const DateRange = styled(
        */
       useUtc: PropTypes.bool,
 
+      /**
+       * handle UTC checkbox change
+       */
+      handleUseUtc: PropTypes.func,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange.jsx]: Use `on*` when it's a callback prop, and `handle*` for the function/method that handles a callback.

File: src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx [hunk 2]
```diff
@@ -186,30 +182,34 @@ const SelectorItem = styled(
...
   cursor: pointer;
   white-space: nowrap;
-  padding: ${space(1)} ${space(2)};
+  padding: ${space(1)};
   align-items: center;
-  border-bottom: 1px solid ${p => p.theme.borderLight};
+  flex: 1;
+  background-color: ${p => (p.selected ? p.theme.offWhite : 'transpatent')};
+  font-weight: ${p => (p.selected ? 'bold' : 'normal')};
+  border-bottom: 1px solid ${p => (p.last ? 'transparent' : p.theme.borderLight)};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx]: Would `:last-child` work?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx]: last child is kinda poorly implemented and I would prefer we avoid it unless there is a situation where we truly want to apply a style to anything appended to the end of the parent _no matter what it is_ (but in my experience this situation never occurs)

File: src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange.jsx (hunk 1 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange.jsx]: 🙏

**Pull request merged**